### PR TITLE
[Release] Merge develop 1.0.0 to master

### PR DESCRIPTION
**Pull request by aappoalander**
## Description
Merge most recent changes to master and publish the DS-site.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "suomifi-design-system",
   "private": true,
   "description": "Suomi.fi Design System",
-  "version": "0.2.0",
+  "version": "1.0.0",
   "author": "Jari Porola <jari.porola@gmail.com>",
```
Comment (ketsappi) [inline package.json:6]: Would this be good place to update also author/contributors for the repo?

Comment (aappoalander) [reply package.json:6]: Not at this point. During the next change maybe as this PR is only merge from dev to master.

**Review by ketsappi — APPROVED** [review]
Tested locally and visited every page and seems to work as intended 👍

**Review by LJKaski — APPROVED** [review]
Seems to be working and looking like it should :)

**Pull request merged**